packages/react-dom/src/__tests__/ReactTestUtilsAct-test.js

```diff
@@ -31,476 +55,502 @@ describe('ReactTestUtils.act()', () => {
     ReactDOM = require('react-dom');
     ReactTestUtils = require('react-dom/test-utils');
     SchedulerTracing = require('scheduler/tracing');
+    ReactFeatureFlags = require('shared/ReactFeatureFlags');
     act = ReactTestUtils.act;
     container = document.createElement('div');
     document.body.appendChild(container);
   });
   afterEach(() => {
-    ReactDOM.unmountComponentAtNode(container);
+    unmount(container);
     document.body.removeChild(container);
   });
 
-  describe('sync', () => {
-    it('can use act to flush effects', () => {
-      function App(props) {
-        React.useEffect(props.callback);
-        return null;
-      }
-
-      let calledLog = [];
-      act(() => {
-        ReactDOM.render(
-          <App
-            callback={() => {
-              calledLog.push(calledLog.length);
-            }}
-          />,
-          document.createElement('div'),
-        );
-      });
-
-      expect(calledLog).toEqual([0]);
+  describe('mocked scheduler', () => {
+    beforeEach(() => {
+      ReactFeatureFlags.warnAboutMissingMockScheduler = true;
+      // todo - this doesn't actually work
```

Comment:
@acdlite I'm kinda struggling here - nothing I'm trying is able to unmock the mocked scheduler. does anything here (in the commented code) look out of place to you? plsthx!

Reply:
After unmocking it, you need to re-require the modules. The variables still point to the old versions.